src/data/builder-tools.js

```diff
@@ -770,6 +770,14 @@ export const Showcases = [
     getstarted: "/docs/get-started/aiken",
     tags: ["favorite", "cli", "plutus"],
   },
+  {
+    title: "Acca",
+    description: "Library extending Aiken's standard library",
+    preview: require("./builder-tools/aiken.png"),
+    website: "https://github.com/Cardano-Fans/acca",
+    getstarted: null,
+    tags: ["favorite", "plutus", "library", "aiken"],
```

Comment:
Reviewers: what's the feeling on adding the currently nonexistent `aiken` tag?  (I'll do it in this PR if acceptable.)  Right now the only other entry I'm sure it can be applied to is Aiken itself... which seems a little too thin unless other libraries & dev tools are expected.

@matiwinnetou the term `favorite`, to use your language from the GitHub page, is "quite opinionated what is should be in there".  I'd never heard of `acca` before so keeping the tag `favorite` in this list should be endorsed by 1 or 2 other editors.

Reply:
I will remove term favourite.

Reply:
> other libraries & dev tools are expected

I think they're expected 🙂

Reply:
@rdlrt should we strike while iron is hot then & add the `aiken` keyword to this PR?

Reply:
Ye - why not 🙂

Reply:
OK, I'll cook it up 🤠

Reply:
@matiwinnetou looks like we don't have write access to your branch!  @rdlrt I'll watch for when this one is merged & then add the `aiken` keyword & apply it in another PR afterward.